pkgs/servers/alps/default.nix

```diff
@@ -15,10 +15,26 @@ buildGoModule rec {
 
   proxyVendor = true;
 
+  outputs = [
+    "out"
+    "public"
+  ];
+
+  ldflags = [
+    "-X main.themesPath=${placeholder "public"}/themes"
```

Comment:
OT, but why do we set this on every single package with ld flags? Seems like something `buildGoModule` should take care of rather than every author having to remember this.

Reply:
Adding `-s -w` by default would break some programs, see https://github.com/NixOS/nixpkgs/issues/177698